src/app/core/component.ts

```diff
@@ -532,9 +536,9 @@ export default class Component {
      * @memberof Component
      */
     getRotation() {
-        if (this.__params.hasParam("rotation")) {
+        if (this._params.hasParam("rotation")) {
             return this.getValue("rotation");
-        } else if (this.__params.hasParam("orientation")) {
+        } else if (this._params.hasParam("orientation")) {
             const orientation = this.getValue("orientation");
             if (orientation === "V") {
                 return 0;
```

Comment:
Let get rid of this orientation parameter check. this is once again for legacy components